parser_library/src/workspaces/workspace.cpp

```diff
@@ -529,7 +535,7 @@ void workspace::find_and_add_libs(
 // open config files and parse them
 bool workspace::load_and_process_config()
 {
-    std::filesystem::path ws_path(uri_);
+    std::filesystem::path ws_path(location_.get_path());
```

Comment:
Use the new reading API?

Reply:
If it is ok with you I would return to this in some of the next PRs alongside the doc update and library selection by patterns.

Reply:
ok